### ISSUE TITLE
AutoIncrement column value cannot be set.

### ISSUE THREAD

**Issue by jiriadamec** [Raizlabs/DBFlow#1447]
DBFlow Version: 4.1.1
Issue Kind (Bug, Question, Feature): Bug

Description: 
When I set a value to autoincrement column and try to save the model an exception occurs.
I created sample that simulates the error:

```
@Table(database = TestDb.class) 
public class TestModel extends BaseModel {

	@PrimaryKey(autoincrement = true)
	private int modelId;
	@Column
	private String name;


	public int getModelId() {
		return modelId;
	}


	public void setModelId(int modelId) {
		this.modelId = modelId;
	}


	public String getName() {
		return name;
	}


	public void setName(String name) {
		this.name = name;
	}
}
```

```
@Test
public void testDb() {
	TestModel testModel = new TestModel();
	testModel.setModelId(1);
	testModel.setName("name");

	testModel.save();

	assertEquals(1, SQLite.select().from(TestModel.class).queryList().size());
}
```

`java.lang.IllegalArgumentException: Cannot bind argument at index 2 because the index is out of range.  The statement has 1 parameters.`

**Comment by jiriadamec** [comment]
I think the problem is in AutoIncrementModelSaver:

```
@Override
    public synchronized long insert(@NonNull TModel model,
                                    @NonNull DatabaseStatement insertStatement,
                                    @NonNull DatabaseWrapper wrapper) {
        if (getModelAdapter().hasAutoIncrement(model)) {
            getModelAdapter().bindToStatement(insertStatement, model);
        } else {
            getModelAdapter().bindToInsertStatement(insertStatement, model);
        }
        long id = insertStatement.executeInsert();
        if (id > INSERT_FAILED) {
            getModelAdapter().updateAutoIncrement(model, id);
            NotifyDistributor.get().notifyModelChanged(model, getModelAdapter(), BaseModel.Action.INSERT);
        }
        return id;
    }
```

the `insertStatement` does not contain `modelId` column, but the adapter tries to bind it.

**Comment by agrosner** [comment]
fixed for `4.1.2`.
